Sluggo-SwiftUI/Views/TicketEditDetail.swift

```diff
@@ -0,0 +1,181 @@
+
+    }
+    
+    var body: some View {
+        NavigationStack {
+            List {
+                Section {
+                    TextField("Title (Required)", text: $ticketTitle)
+                }
+                Section {
+                    ExtendedPicker(title: "Assigned User", items: $teamMembers, selected: $ticketUser)
+                }
+                Section {
+                    ExtendedPicker(title: "Statuses", items: $ticketStatuses, selected: $ticketStatus)
+                }
+                Section {
+                    ZStack{
+                        HStack {
+                            NilContext(item: ticketTags) {
+                                Text((ticketTags.map({$0.getTitle()})).joined(separator: ", "))
+                            } nilContent: {
+                                Text("Tags")
+                            }
+                            .fixedSize(horizontal: false, vertical: true)
+                            Spacer()
+                            Image(systemName: "plus.square")
+                                .foregroundColor(.blue)
+                            }
+                        NavigationLink("Tags") {
+                            MultiSelectionList(items: $ticketAllTags, selected: $ticketTags)
+                                .navigationBarTitle("Tags")
+                                .navigationBarTitleDisplayMode(.inline)
+                        }
+                        .opacity(0)
+                    }
+                }
+                Section {
+                    Toggle("Due Date", isOn: $dateToggle.animation())
+                    if dateToggle {
+                        DatePicker("Date", selection: $ticketDueDate).labelsHidden()
+                        // Toggle throwing strange warning
+                    }
+                }
+                Section() {
+                    TextField("Description", text: $ticketDescription, axis: .vertical)
+                        .frame(minHeight: 100, alignment: .topLeading)
+                }
+            }
+            .navigationBarTitle("Edit Ticket")
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationBarItems(
+                leading: Button("Cancel") {
+                    self.showView.toggle()
+                    
+                }, trailing: Button("Done") {
+                    if ticketTitle.isEmpty {
+                        showAlert = true
+                    } else {
+                        Task.init(priority: .userInitiated) {
+                            await self.doUpdate()
+                        }
+                        self.showView.toggle()
+                    }
+                })
+            .task(doLoad)
+            .alert("Title Required", isPresented: $showAlert) {
+                Button("OK", role: .cancel) { }
+            }
+        }
+    }
+    
+    @Sendable func doUpdate() async {
```

Comment:
If we want to use this view to also deal with Creating tickets, we need to consider if this function is going to handle both cases or just update. More of a discussion rather than a change.

Reply:
I'll see if I can abstract the view a bit more